src/domains/core/Loading/Loading.styles.js

```diff
@@ -20,9 +21,9 @@ export const LoadingBox = styled.div`
     border-radius: 50%;
     content: "";
     height: 60px;
-    left: 50%;
+    left: 45%;
     position: absolute;
-    top: 50%;
+    top: 25%;
```

Comment:
Will this work for all cases where we use the `<Loading />`?

Reply:
Yes 👀